src/Raven.Server/ServerWide/Commands/EditTimeSeriesConfigurationCommand.cs

```diff
@@ -28,5 +31,24 @@ public override void FillJson(DynamicJsonValue json)
         {
             json[nameof(Configuration)] = Configuration.ToJson();
         }
+
+        public override void AssertLicenseLimits(ServerStore serverStore, DatabaseRecord databaseRecord, ClusterOperationContext context)
+        {
+            if (CanAssertLicenseLimits(context, minBuildVersion: MinBuildVersion54200, serverStore) == false)
+                return;
+
```

Comment:
You must load the license with the same context that you are using here (`ClusterOperationContext`). You might inconsistent results if you load the license from the storage.

Reply:
Done